docs/platform-integration/storage/file-system-helpers.md

```diff
@@ -0,0 +1,80 @@
+
+### Cache directory
+
+To get the application's directory to store **cache data**. Cache data can be used for any data that needs to persist longer than temporary data, but shouldn't be data that is required to operate the app, as the operating system may clear this storage.
+
+:::code language="csharp" source="../snippets/shared_1/Storage.cs" id="filesys_cache":::
+
+### App data directory
+
+To get the app's top-level directory for any files that aren't user data files. These files are backed up with the operating system syncing framework.
+
+:::code language="csharp" source="../snippets/shared_1/Storage.cs" id="filesys_appdata":::
+
+## Bundled files
+
+To open a file that is bundled into the app package, use the `OpenAppPackageFileAsync` method and pass the file name. This method returns an <xref:System.IO.Stream> representing the file contents. The following example demonstrates using a method to read the text contents of a file:
+
+:::code language="csharp" source="../snippets/shared_1/Storage.cs" id="filesys_readtxtfile":::
+
+## Platform differences
+
+This section describes the platform-specific differences with the file system helpers.
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+- `FileSystem.CacheDirectory`\
+Returns the [CacheDir](https://developer.android.com/reference/android/content/Context.html#getCacheDir) of the current context.
+
+- `FileSystem.AppDataDirectory`\
+Returns the [FilesDir](https://developer.android.com/reference/android/content/Context.html#getFilesDir) of the current context, which are backed up using [Auto Backup](https://developer.android.com/guide/topics/data/autobackup.html) starting on API 23 and above.
+
+- `FileSystem.OpenAppPackageFileAsync`\
+Add any file into the _Assets_ folder in the Android project, and mark the **Build Action** as **AndroidAsset** to use it with `OpenAppPackageFileAsync`.
+
+# [iOS](#tab/ios)
+
+- `FileSystem.CacheDirectory`\
+Returns the [Library/Caches](https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html) directory.
+
+- `FileSystem.AppDataDirectory`\
+Returns the [Library](https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html) directory that is backed up by iTunes and iCloud.
+
+> [!IMPORTANT]
+> In the iOS Simulator, the Application ID (which is part of the directory name) changes on every build so you have to retrieve the correct ID each time you build your application for the Simulator.
+
+- `FileSystem.OpenAppPackageFileAsync`\
+Add any file into the _Resources_ folder in the iOS project, and mark the **Build Action** as **BundledResource** to use it with `OpenAppPackageFileAsync`.
```

Comment:
As above (see https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-mobile/Resources/Raw/AboutAssets.txt)

So change the build action and folder location, and say it's in the app project (aka single project).